pkg/version/version.go

```diff
@@ -0,0 +1,26 @@
+// Package version contains the version information of the krew binary.
+package version
+
+var (
+	// GitCommit contains the git commit idenifier.
+	GitCommit string
+
+	// GitTag contains the git tag or describe output.
+	GitTag string
```

Comment:
As you said, I think we don't need the tag. It is not useful to the user.
The only thing needed for devs is the commit.

Reply:
On the contrary, we actually need only the tag. As a dev I can figure out the commit id from the sha. But as a user I can’t find out vX.Y.Z easily from the sha OR commitID.

Reply:
I agree that the tag is useful for the User.

We cannot figure out the CommitID from the zip sha. CommitID is for developers, it points to a specific version in time. The tag may point to a past point in time.

Reply:
Let's keep both, it's not an API.